test/networking/rpc_client_test.cpp

```diff
@@ -28,59 +27,64 @@ namespace test {
 
 class RpcClientTests : public PelotonTest {};
 
-/*
 TEST_F(RpcClientTests, BasicTest) {
-  networking::RpcServer rpc_server(PELOTON_SERVER_PORT);
-  networking::PelotonService service;
-  rpc_server.RegisterService(&service);
+  // FIXME: If we comment out the entire test case, then there'll be some static
```

Comment:
I didn't really fix it. We have to add a blank test to call TearDown at least once. There're some global variables will be initialized even if we do nothing but I don't know what they are.